### PR TITLE
adjust maintainers/admins of kube-openapi

### PR DESCRIPTION
**Pull request by lavalamp**

### CODE REVIEW

**Comment by lavalamp** [comment]
Followup to #1585.

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -1605,13 +1605,15 @@ teams:
     description: Admin access to the kube-openapi repo
     members:
     - apelisse
-    - mbohlool
+    - lavalamp
+    - deads2k
```
Comment (justaugustus) [inline config/kubernetes/org.yaml]: Needs alpha-sort

Comment (lavalamp) [reply config/kubernetes/org.yaml]: fixed, sorry

**Pull request merged**